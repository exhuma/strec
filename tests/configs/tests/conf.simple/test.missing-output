there is no "expected" output for this test
