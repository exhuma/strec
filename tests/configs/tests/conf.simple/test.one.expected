the <blue>quick<reset> brown <red>fox<reset> jumps over the lazy dog.
